deployment_config/sandbox_vars.yml

```diff
@@ -1,6 +1,6 @@
-AUTH_CLIENT_ID: ${AUTH_CLIENT_ID}
-AUTH_CLIENT_SECRET: ${AUTH_CLIENT_SECRET}
+AUTH_CLIENT_ID: ((AUTH_CLIENT_ID))
```

Comment:
@rahearn Do you mind sanity checking this? The deployment appears to work.. and the variables make it into cloud.gov correctly.  It's feeling a bit ♾️  though.

Reply:
You know what, nvm, I'm bolding soldiering on. :) I'll get your feedback on the big AH => HHS merge.